schema-bridge.graphql

```diff
@@ -23,6 +23,67 @@ enum Network {
   OSMOSIS
   MATIC # aka Polygon
   XDAI # aka Gnosis Chain
+  UBIQ
```

Comment:
I don't think there is a need to add any network that's not yet supported by The Graph